decidim-dev/lib/decidim/test/rspec_support/component.rb

```diff
@@ -1,42 +1,31 @@
 # frozen_string_literal: true
-RSpec.shared_context "component" do
-  let!(:feature_manifest) { raise NotImplementedError }
-  let!(:component_manifest) { raise NotImplementedError }
+RSpec.shared_context "feature" do
```

Comment:
Rename file to `feature_context.rb` to differentiate this form the `Feature` module file.